internal/integration/rules_test.go

```diff
@@ -428,6 +428,17 @@ func TestAddAttributesRules(t *testing.T) {
 	}
 }
 
+func TestEmptyIgnoreRules(t *testing.T) {
```

Comment:
could you add the test case that reproduce the bug? It will make easier to understand the exact case

Reply:
I did it already adding `Prefixes: []string{"not_matching"},`. I can create one more specific if needed. Also in the issue there are some info

Reply:
I added some further tests